### ISSUE TITLE
QGIS GeoNode connection returns no layers

### ISSUE THREAD

**Issue by gubuntu** [kartoza/LEDET_BIMS#295]
# problem

I can create a GeoNode connection in QGIS 3 to http://bims.ledet.gov.za but no layers are returned. 

# expected behaviour
all public layers should be returned

**Comment by NyakudyaA** [comment]
@gubuntu Are we sure that Geonode client can  is working ?
![screenshot 2018-10-11 at 17 13 59](https://user-images.githubusercontent.com/2510900/46814533-3cd30780-cd79-11e8-92db-401968fc5b75.png)

@myarjunar can you confirm if the Geonode client is working. I have tried to connect to testing.geonode.kartoza and other services as depicted in the image above and nothing is returned.

**Comment by myarjunar** [comment]
I've tried it and it is working with http://geonode.inasafe.org/ but not with http://testing.geonode.kartoza.com .

**Comment by gubuntu** [comment]
Yes I find that too. The main difference is that inasafe is 2.9 with QGIS and testing is our fork of 2.8 with QGIS.

Perhaps we broke something on the GeoNode side @lucernae? @myarjunar if you think it's a QGIS issue please create a QGIS ticket for it.

**Comment by NyakudyaA** [comment]
@myarjunar Geonode client basically makes an API call to geonode to access the layers from 

http://bims.ledet.gov.za/api/layers/ running the following from the browser connected over vpn I can access the layer list. Even doing filters on the API yields results http://bims.ledet.gov.za/api/layers/?title__contains=biome

![screenshot 2018-10-15 at 12 50 59](https://user-images.githubusercontent.com/2510900/46946735-00edba00-d079-11e8-9504-a8434d91201f.png)

However if I try to access it directly from the python console in QGIS it just hangs and do not get anything returned. Do you think this could be an issue with a slow connection from QGIS which causes the server to timeout and then not get any layer list.

**Comment by gubuntu** [comment]
this is low priority - as long as OWS and PostGIS connections work we're good
